app/ts/components/AmountField.tsx

```diff
@@ -52,10 +52,9 @@ export const AmountField = (props: Props) => {
 		getTokenBalance(address.value.value, props.token.address)
 	}
 
-
 	useSignalEffect(() => {
 		if (tokenBalance.value.state !== 'resolved') return
-		const balance = ethers.utils.formatUnits(tokenBalance.value.value, props.token?.decimals)
+		const balance = formatUnits(tokenBalance.value.value, props.token?.decimals)
```

Comment:
if decimals is undefined, I believe this uses 16 decimals. Is this intended?

Reply:
** 18 decimals, the same as Ether

Reply:
the token decimals and it's data comes from the stored data in localstorage so it should exist prior to saving